content/references/markdown/index.md

```diff
@@ -4,12 +4,171 @@ topic: markdown
 next: /references/markdown/extensions/
 ---
 
+
+
 # Basic Markdown Syntax
 
 Codewars supports [Markdown][wiki-markdown].
 
 More specifically, [GitHub Flavored Markdown][gfm] (strict superset of [CommonMark][common-mark]) and few [Codewars extensions][extensions].
 
+You may find below a summary of the most used ones.
+
+
+
+
+## Emphasis
+
+```markdown
+_This is italic_
+*This is also italic*
+
+__This is bold__
+**This is bold too**
+
+___here, italic and bold___
+***here, italic and bold***
+```
+Note that `*You **can** combine them*`: *You **can** combine them*.
+
+
+
+
+## Headers
+
+<pre><code>
+# This is &lt;h1&gt;
+## This is &lt;h2&gt;
+### This is &lt;h3&gt;
+</code></pre>
+
```

Comment:
```suggestion
```

Let's just remove headers. It's not that useful for writing comments and it's styled like bold text in comments so it's misleading.

We can have a page for writing kata descriptions, and we can mention headers there.